routes/api/contacts-router.js

```diff
@@ -0,0 +1,80 @@
+import express from "express";
+import Joi from "joi";
+
+import contactsService from "../../models/contacts.js";
+import { HttpError } from "../../helpers/index.js";
+
+const router = express.Router();
+
+const contactAddShema = Joi.object({
```

Comment:
schema should be in separate file, create middleware which will use schema to validate req.body